app/views/recipes/public_recipes.html.erb

```diff
@@ -0,0 +1,36 @@
+<p style="color: green"><%= notice %></p>
+
+<div class="page-title">
+  <h1>Public Recipes</h1>
+</div>
+
+<% if @public_recipes.empty? %>
+        <h3>Currently there aren't public recipes in the database</h3>
+      <% else %>
+
+<div class="recipe-list">
+
+  <% @public_recipes.each do |recipe| %>
+   <div class="recipe-card">
+
+    <div class="card-content1">
+  <p>
+     <h3><%= link_to recipe.name, recipe, class:"link" %></h3>
+  </p>
+
+<h4><%= "By #{recipe.user.name}" %></h4>
+
+  </div>
+
+<div class="card-content2">
+
+  <p><strong>Total food items: </strong><%= recipe.recipe_foods.count %></p>
+  <p><strong>Total price ($): </strong><%= number_to_currency(recipe.total_cost) %></p>
+
+  </div>
+
+</div>
+
+  <% end %>
+<% end %>
+<%= link_to "Back to recipes", recipes_path %>
```

Comment:
- [ ] The `total food items` and `total price` of the public recipes are not showing.

Your Design |
:--------------------:|
![Screen Shot 2022-11-10 at 10 15 04 PM](https://user-images.githubusercontent.com/39523312/201218174-f7903d2e-b1b8-4c5e-9be0-424d687273a0.png) |

Wireframe |
:----------------:|
![Screen Shot 2022-11-10 at 10 18 33 PM](https://user-images.githubusercontent.com/39523312/201218000-01f45a85-9d60-4fa0-a4b7-1280521956bd.png) |

Reply:
Okay, I'll fix it, thank you